README.md

```diff
@@ -34,39 +32,7 @@ Supported tags and release links
 * [3.0.2](https://github.com/dtzar/helm-kubectl/releases/tag/3.0.2) - helm v3.0.2, kubectl v1.17.0, alpine 3.10
 * [3.0.1](https://github.com/dtzar/helm-kubectl/releases/tag/3.0.1) - helm v3.0.1, kubectl v1.16.3, alpine 3.10
 * [3.0.0](https://github.com/dtzar/helm-kubectl/releases/tag/3.0.0) - helm v3.0.0, kubectl v1.16.2, alpine 3.10
-* [2.17.0](https://github.com/dtzar/helm-kubectl/releases/tag/2.16.1) - helm v2.17.0, kubectl v1.19.4, alpine 3.12
```

Comment:
Fine with deprecating the full list - would be nice to have a short statement that we've been supporting every version of helm since 2.3.1 in <date>, see releases...